test/unit/controller.js

```diff
@@ -578,7 +578,7 @@ exports['Tessel.get'] = {
         test.equal(this.closeTesselConnections.callCount, 1);
         test.equal(Array.isArray(this.closeTesselConnections.args[0]), true);
         test.equal(this.logsInfo.callCount, 2);
-        test.equal(_.contains(this.logsInfo.lastCall.args[0], 'the_name'), true);
+        test.equal(this.logsInfo.lastCall.args[0].includes('the_name'), true);
```

Comment:
Ahahaha :)

Awesome though, `String.prototype.includes` is supported in 4.0 as well :raised_hands: